packages/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -277,4 +279,123 @@ describe("TinyliciousClient", () => {
         await corruptedAliasOp(runtimeOf(do1), "alias");
         assert(await dataCorruption);
     });
+
+    /**
+     * Scenario: Test if TinyliciousClient with forceWriteMode set to true forces the container to write mode.
+     * Keep the original behaviour otherwise. The flag forceWriteMode will attempt to change the mode to write when
+     * starting a connection, but if the write permission is not available or the file is read only, mode will set
+     * to read.
+     *
+     * Expected behavior: Setting forceWriteMode to true in TinyliciousClientProps should force the container's
+     * connectionMode to `write` while keeping the original behavior if forceWriteMode is not set to true.
+     */
+    it("forces the container to connect with write mode when forceWriteMode is set to true ", async () => {
```

Comment:
This test case seems quite long. Could you break this down in multiple tests (it-s), under a parent item?